### PR TITLE
fingerprint 2019 vw jetta sel

### PR DESCRIPTION
**Pull request by cyanet01**

### CODE REVIEW

**Comment by VirtuallyChris** [comment]
What’s the route this fingerprint came from?

**Comment by cyanet01** [comment]
Hello

186c80a9e18ac816|2021-12-22--22-42-56

That's the route name

II am new to this if not providing correct information
Please tell me where to get the route

Thanks


Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -296,6 +296,7 @@ class CAR:
       b'\xf1\x8704E906024L \xf1\x899970',
       b'\xf1\x8704E906027MS\xf1\x896223',
       b'\xf1\x875G0906259T \xf1\x890003',
+      b'\xf1\x8704E906024C \xf1\x899970',
```
Comment (jyoung8607) [inline selfdrive/car/volkswagen/values.py:299]: Move to 3rd on the list

File: selfdrive/car/volkswagen/values.py [hunk 2]
```diff
@@ -311,6 +312,7 @@ class CAR:
       b'\xf1\x875Q0959655BR\xf1\x890403\xf1\x82\02311170031313300314240011150119333433100',
       b'\xf1\x875Q0959655BR\xf1\x890403\xf1\x82\02319170031313300314240011550159333463100',
       b'\xf1\x875Q0959655CB\xf1\x890421\xf1\x82\x1314171231313500314643021650169333613100',
+      b'\xf1\x875Q0959655AG\xf1\x890338\xf1\x82\x1314171231313500314611011630169333463100',
```
Comment (jyoung8607) [inline selfdrive/car/volkswagen/values.py:315]: Move to 2nd on the list

File: selfdrive/car/volkswagen/values.py [hunk 3]
```diff
@@ -319,10 +321,12 @@ class CAR:
       b'\xf1\x875QM909144C \xf1\x891082\xf1\x82\00521A00642A1',
       b'\xf1\x875QM909144C \xf1\x891082\xf1\x82\00521A10A01A1',
       b'\xf1\x875QN909144B \xf1\x895082\xf1\x82\00571A10A11A1',
+      b'\xf1\x875QM909144B \xf1\x891081\xf1\x82\x0521A10A01A1',
```
Comment (jyoung8607) [inline selfdrive/car/volkswagen/values.py:324]: Remove; duplicate

File: selfdrive/car/volkswagen/values.py [hunk 4]
```diff
@@ -319,10 +321,12 @@ class CAR:
...
     ],
     (Ecu.fwdRadar, 0x757, None): [
       b'\xf1\x875Q0907572N \xf1\x890681',
       b'\xf1\x875Q0907572R \xf1\x890771',
+      b'\xf1\x875Q0907572P \xf1\x890682',
```
Comment (jyoung8607) [inline selfdrive/car/volkswagen/values.py:329]: Move to 2nd on the list

**Review by jyoung8607 — CHANGES_REQUESTED** [review]
Can you help us out by adding these in sorted order? We try to keep them sorted to make it easier to find duplicates in the future.

**Comment by cyanet01** [comment]
Hello

I don’t believe there is any duplicate lines

Richard

**Comment by jyoung8607** [comment]
> Hello I don’t believe there is any duplicate lines 

Your EPS firmware was already on that list, second position. Automated CI checks later in the process will kick back duplicates, they're just not running for you yet as a new contributor.

I took the liberty of filing #23306 with your commit preserved along with the sorting/dedupe changes so we can go ahead and get you supported. Thank you for the data!

**Pull request closed without merging**